ffi/dotnet-idsdk/IoticsIdLib/src/id/IdLib.cs

```diff
@@ -0,0 +1,176 @@
+using System.Runtime.InteropServices;
+
+/**
+Wrapper class that loads the DLL and exposes the methods as C# code. 
+*/
+internal partial class IdLib
+{
+    // Define the struct that matches the return type of CreateDefaultSeed in Go
+    [StructLayout(LayoutKind.Sequential)]
+    public struct Return
+    {
+        public IntPtr r0;
+        public IntPtr r1;
+    }
+
+    // Declare the P/Invoke function
+    [LibraryImport("lib-iotics-id-sdk")]
+    [UnmanagedCallConv(CallConvs = [typeof(System.Runtime.CompilerServices.CallConvCdecl)])]
+    public static partial IntPtr TestEmojy([MarshalAs(UnmanagedType.LPUTF8Str)] string emo);
```

Comment:
To Remove?

Reply:
FYI - the point of this (temporary test) was to check that `LPUTF8Str` would be correct way to correctly get UTF8 from C# to Go and Go to C#.
Since none of the existing functions allow easy bi-directional checks, I suggested this.
(But yes, I agree - to remove - if the test proves it's working.)

Reply:
done